### DISCUSSION TITLE
Using Mamba instead of Conda for FLARE?

### DISCUSSION THREAD

**Discussion by turbosonics** [mir-group/flare#391] [Q&A]
Hello, 

Has anyone tried to install & run FLARE using Mamba instead of Conda? Are there anything I need to modify to prevent different performance or installation errors? I think there shouldn't be, but just in case asking...

Thanks!

**Comment by YuuuXie** [comment]
I did not try mamba before, but I assume it should be the same as conda. You can let me know if there is any issue with it
